### PR TITLE
removing todo-link

### PR DESCRIPTION
**Pull request by LEEDASILVA**

### CODE REVIEW

File: subjects/alphacount.fr.md [hunk 1]
```diff
@@ -17,20 +17,20 @@ func AlphaCount(str string) int {
...
-Here is a possible [program](TODO-LINK) to test your function :
+Here is a possible program to test your function :
 
 ```go
 package main
 
 import (
-	"fmt"
-	piscine ".."
+    "fmt"
```
Comment (xpetit) [inline subjects/alphacount.fr.md]: why convert tabs to spaces ?

Comment (LEEDASILVA) [reply subjects/alphacount.fr.md]: that one was a mistake, thanks :)

File: subjects/btreeprintroot.fr.md [hunk 2]
```diff
@@ -9,8 +9,8 @@ Les nodes doivent être définies comme ci-dessous:
 
 ```go
 type TreeNode struct {
-	left, right *TreeNode
-	data        string
+    left, right *TreeNode
```
Comment (xpetit) [inline subjects/btreeprintroot.fr.md]: What about this one ? 😅

Comment (LEEDASILVA) [reply subjects/btreeprintroot.fr.md]: good eye XD, thanks

Comment (LEEDASILVA) [reply subjects/btreeprintroot.fr.md]: will take a better look

**Pull request merged**